pyproject.toml

```diff
@@ -39,15 +39,17 @@ teamcity-messages = ">=1.31"
 wheel = "~=0.40.0"
 
 # Dev tools
-black = "~=24.3"
```

Comment:
Is this a good idea to open this to any greater version?

Reply:
I don't feel like it is a problem, respecting the fact that we are installing dependencies using lock file